app/api/stores/router.py

```diff
@@ -21,3 +21,17 @@ def update_store(
     updated_store = service.update_store(database, store.id, payload)
 
     return updated_store
+
+
+@router.get("/{store_id}/items", response_model=list[schemas.SearchItem])
+def search_store(
+    keyword: str = "%",
+    store: models.Store = Depends(validate_store_id),
+    database: Session = Depends(get_db)
+):
+    search_item = service.search_item_by_keyword(database, store.id, keyword)
+
+    if keyword == "":
+        return []
+
+    return search_item
```

Comment:
```suggestion
    items = service.search_items_by_keyword(database, store.id, keyword)

    if keyword == "":
        return []

    return items
```

變數命名以名詞開頭

可能會找到「多個」商品，所以要命名為 `search_items_by_keyword`